README.md

```diff
@@ -1 +1,107 @@
-# My-Portfolio
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+**[Portfolio]** is a beginning to a long interesting journey i'm about to take with Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>CSS</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+- **Setting up mobile portfolio**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+
+### Setup
+
+Clone this repository to your desired folder: https://github.com/denigit5/Portfolio.git
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+👤 **Author**
+
+- GitHub: [@denigit5](https://github.com/denigit5)
+- Twitter: [@denysemukiza](https://twitter.com/denyse_mukiza)
+- LinkedIn: [Denyse Mukiza](https://www.linkedin.com/in/denyse-mukiza-92a1b0228/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Live Demo]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project don't hasitate to contact me or use the features present.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse and the team for putting their time and effort to make this happen.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [0ptional] I will mark this suggestion as optional because this was not mentioned to you previously, kindly work on your readme to make it more descriptive and professional All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in a student's repo
- [ ] Your Readme has no License Mit please kindly fix this kindly fix it for a professional reason The MIT license gives express permission for users to reuse code for any purpose, sometimes even if the code is part of proprietary software. 
- [ ] Professional Readme
![PREFESSIONAL Readme](https://github.com/denigit5/Portfolio/assets/92458236/8fd7a1d5-a341-43c2-af8e-03c26f7c8ce2)

- [ ] You Readme
![chrome-capture-2023-6-8](https://github.com/denigit5/Portfolio/assets/92458236/c2305a77-b6e7-4afc-87b4-cd65ae79768a)